admin/services/school-summary.service.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+const dataService = require('./data-access/school-summary.data.service')
+
+const service = {}
+
+/**
+ * @description get check activity summary
+ * @param {number} schoolId
+ * @return {Promise<object>}
+ */
+service.getSummary = async (schoolId) => {
```

Comment:
I think we shouldn't use anonymous functions as it would make any use of APM later on unreadable.

Reply:
done for service and data service